### PR TITLE
Set up Python REST API over the local MySQL server

### PR DESCRIPTION
**Pull request by iprunache**
## Why
We want to provide a common API over the local data store that every CAMI service can address directly.

## What
- [x] set up Django ORM
- [x] set up tastypie over Django
- [x] write Ansible Plyabook for provisioning the API
- [x] implement simple endpoint to validate the API

### CODE REVIEW

**Comment by iprunache** [comment]
@candrei @aismail please review this if you have the time.

File: ansible/roles/django/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+- block:
+    - name: Install required packages
+      apt: pkg={{ item }} state=installed update-cache=yes
+      with_items:
+       - libmysqlclient-dev
+
+    - name: Install python requirements
```
Comment (candrei) [inline ansible/roles/django/tasks/main.yml]: Why needed since `ansible/medical_compliance.yml` installs them from `requirements.txt`?

**Comment by candrei** [comment]
@iprunache really nice Ansible design (with roles) :+1:

File: ansible/roles/django/tasks/main.yml (hunk 1 above)
Comment (iprunache) [inline ansible/roles/django/tasks/main.yml]: The intention was to make `django` a role that independently sets up a Django install that can be used by any playbook. The truth is that the role has a mix of frameworks which are not strictly needed for a simple Django app(tastypie, gunicorn, swagger). I'll just leave the Django dependency here for now.

**Comment by candrei** [comment]
:+1:

**Pull request merged**